epub33/core/index.html

```diff
@@ -2325,17 +2325,32 @@ <h6>The <code>link</code> Element</h6>
 							<aside class="example">
 								<p>The following example shows a reference to a metadata record embedded in a
 										<code>script</code> element inside an <a>XHTML Content Document</a>. Note that
-									the media type of the embedded record is obtained from <code>type</code> attribute
-									on the <code>script</code> element; it is not specified in the <code>link</code>
-									element.</p>
+									the media type of the embedded record (i.e., <code>application/ld+json</code>) is 
+									obtained from <code>type</code> attribute on the <code>script</code> element; 
```

Comment:
Minor typo in the original text: s/obtained from `type` attribute/obtained from the `type` attribute/

Reply:
Handled in https://github.com/w3c/epub-specs/pull/1545/commits/1c0fdf8cdd408c1d6e0bc49677f6f038c03ab41b